flake.nix

```
@@ -0,0 +1,61 @@
+{
+  description = "Build a cargo project without extra checks";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
```

For _some_ stability

```suggestion
    nixpkgs.url = "nixpkgs/nixos-23.11";
```